emails/templates/emails/wrapped_email.html

```diff
@@ -57,6 +57,11 @@
                     <br><a href="{{ survey_link }}" target="_blank" style="font-family: sans-serif; color: #20123a; text-decoration: underline; font-weight: bolder; font-size: 13px;">{{ survey_text }}</a>.
                     {% endif %}
                   {% endif %}
+                  {% if recipient_profile.has_premium %}
+                    <br />
+                    <img alt="" width="24" style="display: inline-block; margin-bottom: 0px; width: 24px;"  src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAAACXBIWXMAAA7DAAAOwwHHb6hkAAAA GXRFWHRTb2Z0d2FyZQB3d3cuaW5rc2NhcGUub3Jnm+48GgAAAX1JREFUSIntlLEvQ2EUxX/3a+uN JcZuXRm6EIn4AyTWSqh0a5sOIozCPyFEWhual7B2EQtRQ1kk5tpsTS0MbT3X4BHE1+cZMPRMN985 95x77/BBH38NCRJsTKvjDJNDWQBGARSuRagMtSnPHkrnxwHbWU0YjyqQskiuniLMFHfl1uZhek3+ Zi40RJmLeiSiHgkV5oEbIGU8qgdpHbD5RG2Ef5YUQgOPibwrzXe0W5rTYwx1IHXnkAc2Q23g3xyF 9cJHcwAKrjQVVgFUydhs7AEwAhB75NQmcGIcAYivDRvwANCNMGgTdNsk/PL+JwEnAEbI2gRqXjjx taECjLAFoMrK9ryOf+b9t2VAPV60oQJy+3KmsAPEjOCW0hp/5UppjRvBBWJAuViR89ABAJ0Wi8Al kFSHtdd3v04KXLRbLPXyCPwqyhkdU7iwjDdR2JN6r/6eGwDkK3KJUPtitFqQ+a8g8EQA5YzWFCY/ ddYK+zIV1Bt4IgAV9Du6Pv4nngE8HXRqmKzo6wAAAABJRU5ErkJggg==" />
```

Comment:
Follow-up: should we add `alt` text here for accessibility?

Reply:
The `i` icon doesn't convey any extra information, so an empty alt text is appropriate here IMHO.